plugins/lookup/hashi_vault.py

```diff
@@ -119,6 +119,12 @@
       description: Controls verification and validation of SSL certificates, mostly you only want to turn off with self signed ones.
       type: boolean
       default: True
+    skip_certificate_validation:
+      description: Controls verification and validation of SSL certificates, mostly you only want to turn off with self signed ones. This takes precedence over validate_certs
+      type: boolean
+      env:
+        - name: VAULT_SKIP_VERIFY
+          version_added: 1.2.1
```

Comment:
```suggestion
          version_added: 1.3.0
```
Patch releases do not get new features (https://semver.org/).